hw/net/pvrdma/pvrdma_backend.c

```diff
@@ -498,6 +498,7 @@ int backend_qp_state_rtr(BackendDevice *dev, BackendQP *qp, u8 qp_type,
         attr.ah_attr.grh.hop_limit  = 1;
         attr.ah_attr.grh.dgid       = ibv_gid;
         attr.ah_attr.grh.sgid_index = gid_idx;
+        attr.rq_psn = rq_psn;
```

Comment:
It is either me or you that are mistaken here but since i was the first then you must stick to my style, make the '=' below the above '='